sips/sip-032/sip-032-improved-stacking.md

```diff
@@ -0,0 +1,232 @@
+# Preamble
+
+**SIP Number:** 032
+
+**Title:** Improved Stacking Protocol
+
+**Authors:**
+
+- Friedger Müffke ([friedger@ryder.id](mailto:friedger@ryder.id))
+
+**Consideration:** Technical
+
+**Type:** Consensus
+
+**Status:** Draft
+
+**Created:** 2025-04-01
+
+**License:** BSD 2-Clause
+
+**Sign-off:**
+
+**Discussions-To:**
+
+- [Stacks Forum Discussions](https://forum.stacks.org/t/remove-cool-down-cycle-in-stacking/17899)
```

Comment:
```suggestion
- https://forum.stacks.org/t/remove-cool-down-cycle-in-stacking/17899
```
style nit, this is not defined in sip-000 so it may be accepted or rejected and resolved.